magma_access_gateway_installer/__init__.py

```diff
@@ -2,10 +2,51 @@
 # Copyright 2022 Canonical Ltd.
 # See LICENSE file for licensing details.
 
-from .agw_installer_ubuntu import AGWInstallerUbuntu
+""" The Access Gateway (AGW) provides network services and policy enforcement. In an LTE network,
+  the AGW implements an evolved packet core (EPC), and a combination of an AAA and a PGW. It works
+  with existing, unmodified commercial radio hardware.
+  For detailed description visit https://docs.magmacore.org/docs/next/lte/architecture_overview.
+"""
```

Comment:
Do we wanna keep AGW's description here? It might seem this is a piece of code of AGW itself, instead of the installer. What do you think?

Reply:
Yeah, it's in README and snapcraft.yaml anyway, so no need to keep it.
Removed.